helsingborg-stad/aterbruket-webapp
# Feat/borrow permission status

Added new status "borrowPermissionDenied".
How to test: 
Create a new borrow advert and restrict by selecting some "förvaltningar". 
Log in with a user from another förvaltning and then the user should not have borrow access. 
Excuse the svengelska. 

![Screenshot 2022-02-07 at 15 39 18](https://user-images.githubusercontent.com/21363149/152811514-f0bb566a-7a2c-4d37-88db-984fad6685a9.png)
